### DISCUSSION TITLE
Considering maintaining a changelog?

### DISCUSSION THREAD

**Discussion by junqdu** [asmyshlyaev177/react-horizontal-scrolling-menu#150] [Ideas]
@asmyshlyaev177 would be cool if there is a changelog or release note for this repo.

**Comment by asmyshlyaev177** [comment]
True, very lazy to do it manually, need to implement something like [commitizen](https://github.com/commitizen-tools/commitizen) for generate it automatically.

**Comment by asmyshlyaev177** [comment]
Tried to configure it, followed this [article](https://blog.logrocket.com/automatically-generate-and-release-a-changelog-with-node-js/).

UPD: seems like it works. Will use.
